O365.Security.Native.ETW.Debug.nuspec

```diff
@@ -1,24 +1,24 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
     <metadata>
-        <id>O365.Security.Native.ETW.Debug</id>
+        <id>Microsoft.O365.Security.Native.ETW.Debug</id>
         <version>1.0.13</version>
```

Comment:
We should bump this to 2.0. By changing the root namespace, it's a breaking change for users.

Reply:
Fixed